components/ItemDetails.js

```diff
@@ -0,0 +1,92 @@
+import styled from "styled-components";
+import Link from "next/link";
+
+import CloseLink from "@/components/CloseLink";
+
+export default function ItemDetails({
+  title,
+  description,
+  initialStatus,
+  isFound,
+  userName,
+  onHandle,
+}) {
+  return (
+    <DetailsWrapper>
+      <Container>
+        <Category initialStatus={initialStatus} isFound={isFound}>
+          {isFound ? "Waiting for pick-up" : initialStatus ? "Lost" : "Found"}
+        </Category>
```

Comment:
It looks like your page hierarchy could be better. This is an h2 element and the highest heading of this component. I would have expected the item title to act as the heading for this component and the category to be more of an additional information. I could be wrong here, it's about the meaning and semantics of your component.